ui/src/components/Modals/OasModal/OasModal.tsx

```diff
@@ -69,7 +70,7 @@ const OasModal = ({ openModal, handleCloseModal }) => {
                   value={selectedServiceName}
                   onChange={onSelectedOASService}
                 >
-                  {oasServices.map((service) => (
+                  {oasServices?.map?.((service) => (
```

Comment:
why? I can't understand when oasServices might be null

Reply:
Happened because UI feature flag was true while server feature flag was false. shouldn't happen normally.